src/test/java/org/openrewrite/java/logging/slf4j/LoggersNamedForEnclosingClassTest.java

```diff
@@ -119,4 +120,22 @@ void method() {
           )
         );
     }
+
+    @Issue("https://github.com/openrewrite/rewrite-logging-frameworks/issues/101")
+    @Test
+    void doNotChangeGetClassOnNonFinalClass(){
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import org.slf4j.Logger;
+              import org.slf4j.LoggerFactory;
+              class WrongClass {}
```

Comment:
I don't think we need WrongClass here right? Or did I miss something?
```suggestion

```

Reply:
nope, I copy pasted it from another test where it was not needed to be consistent. When taking a closer look it's not consistent at all and I cleaned up a bit.